google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerOptions.java

```diff
@@ -1391,6 +1395,24 @@ public Builder setEnableExtendedTracing(boolean enableExtendedTracing) {
       return this;
     }
 
+    /**
+     * Enable spanner server side tracing. Enabling this option will create the trace
+     * spans at the Spanner layer. By default, server side tracing is disabled.
+     */
+    public Builder enableServerSideTracing() {
```

Comment:
Also add an option for enabling via Environment variable, similar to `SPANNER_ENABLE_API_TRACING`

Reply:
Done.